src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,39 @@ public boolean isPrivate() {
         return isPrivate;
```

Comment:
Where did you use these new class? In Address.java you must use these class instead of using a String variable.